docs/security/hardening-octopus.md

```diff
@@ -133,6 +134,77 @@ Add-MpPreference -ExclusionPath "C:\Octopus\Work"
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.0\Server" -name "Enabled" -value "0" -PropertyType "DWord" -Force | Out-Null
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.0\Server" -name "DisabledByDefault" -value 1 -PropertyType "DWord" -Force | Out-Null
+New-Item "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.0\Client" -Force | Out-Null    
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.0\Client" -name "Enabled" -value "0" -PropertyType "DWord" -Force | Out-Null
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.0\Client" -name "DisabledByDefault" -value 1 -PropertyType "DWord" -Force | Out-Null
+
+# TLSv1.1
+Write-Output "Disable TLS 1.1"
+
+New-Item "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Server" -Force | Out-Null
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Server" -name "Enabled" -value "0" -PropertyType "DWord" -Force | Out-Null
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Server" -name "DisabledByDefault" -value 1 -PropertyType "DWord" -Force | Out-Null
+New-Item "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Client" -Force | Out-Null
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Client" -name "Enabled" -value "0" -PropertyType "DWord" -Force | Out-Null    
+New-ItemProperty -path "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Client" -name "DisabledByDefault" -value 1 -PropertyType "DWord" -Force | Out-Null
+```
+
+:::hint
+Once the version of TLS is set, reboot your Server and it should be available via `TLSv1.2`.
+:::
+
+#### Disable SSLv3, TLS 1.0 and 1.1 on Ubuntu Server  {#disable-weak-tls-protocols-ubuntu}
+
+On Ubuntu `20.04` using OpenSSL `1.1.1f` (the latest at time of writing), you can specify the minimum TLS version to use to be `TLSv1.2` by setting the `MinProtocol` directive in the `/etc/ssl/openssl.cnf` OpenSSL config file:
+
+```text
+[system_default_sect]
+MinProtocol = TLSv1.2
+```
+
+On Ubuntu `18.04`, if the `MinProtocol` directive doesn't work, you can try this alternative. When using OpenSSL `1.1.1` (the latest at time of writing), you can specify the available TLS Protocols explicitly in the `/etc/ssl/openssl.cnf` OpenSSL config file:
+
+```text
+[system_default_sect]
+Protocol = -SSLv3, -TLSv1, -TLSv1.1, TLSv1.2
+```
+
+:::hint
+Once the version of TLS is set in your config, you'll want to restart any Tentacle service, and it should be available via `TLSv1.2`.
+:::
+
```

Comment:
I think it would be a good idea to include another `:::hint` at the end of this section telling customers how they can verify that they have successfully removed particular cipher suites. Tools such as `openssl`, and `nmap` can be used and web sites such as [https://www.ssllabs.com/ssltest/](https://www.ssllabs.com/ssltest/) can be used for this purpose.

Reply:
Like it, I'll add that in today thanks @cvandal

Reply:
@cvandal - just for a visual :) 
![image](https://user-images.githubusercontent.com/1418993/151333393-3122c77b-b14e-45df-88ff-9fc532328ef3.png)